### PR TITLE
Timestamp format info update 

### PR DESCRIPTION
**Pull request by ishachopra0401**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by ameypv-rudder — APPROVED** [review]
LGTM

File: rudderstack-api/http-api.mdx [hunk 1]
```diff
@@ -163,7 +163,7 @@ http -a <source_write_key>: <DATA_PLANE_URL>/v1/identify < identify.json
 | `userId` | String | Required, if `anonymousId` is not present. | Unique identifier for a particular user in your database. |
 | `context` | Object | Optional | Dictionary of information that provides context about a message. However, it is not directly related to the API call. |
 | `integrations` | Object | Optional | A dictionary containing the destinations to be either enabled or disabled. |
-| `timestamp` | Date | Optional | The timestamp of the message's arrival. |
+| `timestamp` | Date | Optional | The timestamp of the message's arrival. If you are passing the timestamp in the event, make sure it conforms to the ISO 8601 date format `yyyy-MM-ddTHH:mm:ss.SSSZ`. For example: `2022-02-01T19:14:18.381Z`|
```
Comment (desusai7) [inline rudderstack-api/http-api.mdx:166]: Can we change the Datatype here to `DateTime` from `Date` ?

Comment (desusai7) [reply rudderstack-api/http-api.mdx:166]: As the value includes both date and time in it

Comment (ishachopra0401) [reply rudderstack-api/http-api.mdx]: done

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**